### PR TITLE
@damassi => [Search] Allow pagination to be set from URL, and modify URL on artists tab

### PR DESCRIPTION
**Pull request by mzikherman**
Depends on https://github.com/artsy/metaphysics/pull/1634

Pretty simple!

### CODE REVIEW

File: src/Apps/Search/Routes/Artists/SearchResultsArtists.tsx [hunk 1]
```diff
@@ -15,14 +16,24 @@ export interface Props {
...
+  page: number
+}
+
 const PAGE_SIZE = 10
 
-export class SearchResultsArtistsRoute extends React.Component<
-  Props,
-  LoadingAreaState
-> {
+export class SearchResultsArtistsRoute extends React.Component<Props, State> {
```
Comment (mzikherman) [inline src/Apps/Search/Routes/Artists/SearchResultsArtists.tsx]: Basically this adds the `page: $page` support for fetching the right data based on the incoming URL params, and then we added a little bit of local state to keep track of that for subsequent URL updates.

**Comment by mzikherman** [comment]
Once this and the MP PR are merged, I can add this to the other tabs.

This means that pages like: http://www.artsy.net/search2/artists?term=andy&page=3 wind up working as expected

**Comment by artsyit** [comment]
Deploy preview for *artsy-reaction* ready!

Built with commit 3c38ef76c3adfdebc71f713bac950b5cbc2e6dfd

https://deploy-preview-2224--artsy-reaction.netlify.com

**Comment by damassi** [comment]
LGTM 👍

**Comment by artsyit** [comment]
:rocket: PR was released in v15.11.5 :rocket:

**Pull request merged**